src/vsr/README.md

```diff
@@ -172,7 +176,8 @@ During repair, missing/damaged prepares are requested & repaired chronologically
 In response to a `request_prepare`:
 
 - Reply the `command=prepare` with the requested prepare, if available and valid.
-- Reply `command=nack_prepare` if the request origin is the primary of the ongoing view-change and we never received the prepare. (This enables the primary to truncate uncommitted messages and remain available).
+- Reply `command=nack_prepare` if the request origin is the primary of the ongoing view-change and we never received the prepare.
```

Comment:
:thinking: could we in theory remove `command=nack_prepare` altogether, and just reply with a DVC?